rllib/algorithms/algorithm_config.py

```diff
@@ -406,6 +400,21 @@ def __init__(self, algo_class=None):
         self.seed = None
         self.worker_cls = None
 
+        # `self.fault_tolerance()`
+        self.ignore_worker_failures = False
+        self.recreate_failed_workers = False
+        # By default restart failed worker a thousand times.
+        # This should enough to handle normal transient failures.
+        # This also prevents infinite number of restarts in case
+        # the worker or env has a bug.
+        self.max_num_worker_restarts = 1000
+        # Small delay between worker restarts.
```

Comment:
Explain when users would care about this delay? For example if restarting the env is expected to take more than the default update the number.

Reply:
done